src/msw/graphicsd2d.cpp

```diff
@@ -2118,6 +2118,8 @@ wxD2DFontData::wxD2DFontData(wxGraphicsRenderer* renderer, ID2D1Factory* d2dFact
         L"en-us",
         &m_textFormat);
 
+    wxUnusedVar(hr);
```

Comment:
I don't think it's a good idea to suppress the warning about this variable being unused. It should be used, i.e. we should actually check for error here. Until we can find time to do it, let's leave the warning as a reminder.